speed_python/templates/about.html

```diff
@@ -68,12 +68,12 @@ <h4>The Hosting</h4>
 <h4>Resources</h4>
 <ul>
     <li>
-        <a href="http://mail.python.org/mailman/listinfo/speed" target="_blank">The mailing list</a>
+        <a href="https://mail.python.org/mailman/listinfo/speed" target="_blank">The mailing list</a>
```

Comment:
The list moved to https://mail.python.org/mailman3/lists/speed.python.org/ : can you please update the link?

Reply:
Done.